ui-v2/app/styles/components/composite-row/skin.scss

```diff
@@ -0,0 +1,18 @@
+%composite-row {
+  list-style-type: none;
+}
+%composite-row a {
+  border-top-color: $gray-200;
+  border-right-color: transparent;
+  border-left-color: transparent;
+}
```

Comment:
I was wondering if the last child might need a bottom border also here 🤔 , not sure if it would clash with the footer border or not? What do you think?